packages/template-retail-react-app/app/pages/cart/index.test.js

```diff
@@ -232,32 +255,44 @@ describe('Update quantity', function () {
         )
 
         const editCartButton = within(cartItem).getByRole('button', {name: 'Edit'})
-        userEvent.click(editCartButton)
-        const productView = screen.getByTestId('product-view')
-        expect(productView).toBeInTheDocument()
-        // update item quantity
-        expect(await within(cartItem).getByDisplayValue('2'))
+        await act(async () => {
+            userEvent.click(editCartButton)
+        })
 
-        const incrementButton = await within(cartItem).findByTestId('quantity-increment')
+        const productView = screen.queryByTestId('product-view')
 
-        // update item quantity
-        fireEvent.pointerDown(incrementButton)
+        await act(async () => {
+            const incrementButton = await within(productView).findByTestId('quantity-increment')
+            // update item quantity
+            fireEvent.pointerDown(incrementButton)
+            expect(within(productView).getByDisplayValue('3'))
 
+            const updateCartButtons = within(productView).getAllByRole('button', {name: 'Update'})
+            userEvent.click(updateCartButtons[0])
+        })
+        await waitFor(() => {
+            expect(productView).not.toBeInTheDocument()
+        })
         await waitFor(() => {
             expect(within(cartItem).getByDisplayValue('3'))
         })
+
+        await waitFor(() => {
+            expect(screen.queryByTestId('loading')).not.toBeInTheDocument()
+        })
     })
 })
 
 describe('Remove item from cart', function () {
+    jest.setTimeout(30000)
```

Comment:
it looks like `jest.setTimeout` doesn't work in describe scope... https://stackoverflow.com/a/72640470 It's a global config and not describe specific config.

Reply:
I'm afraid this is misleading and cause people to think "oh, the tests in this describe has a timeout of 30s" which is incorrect